docs/admin/user-management.rst

```diff
@@ -75,23 +205,30 @@ statement returns an error::
     RoleAlreadyExistsException[Role 'Custom User' already exists]
 
 
+.. hide:
+
+    cr> DROP USER "Custom User";
+    DROP OK, 1 row affected (... sec)
+
+
 ``ALTER USER``
 ==============
 
 To alter the password for an existing user from the CrateDB database cluster use
-the :ref:`ref-alter-user` SQL statement::
+the :ref:`ref-alter-role` or :ref:`ref-alter-user` SQL statements::
 
     cr> ALTER USER user_a SET (password = 'pass');
     ALTER OK, 1 row affected (... sec)
 
 The password can be reset (cleared) if specified as ``NULL``::
 
-    cr> ALTER USER user_a SET (password = NULL);
+    cr> ALTER ROLE user_a SET (password = NULL);
```

Comment:
I tried with the latest nightly and this still seems to work with `ALTER USER`?

Reply:
I have one example with `USER` and one with `ROLE`, they are the same, just currently we don't allow to alter a role's password, so:
```
ALTER ROLE role_a SET (password = NULL);
```
is not allowed.

Reply:
Ok, we could leave it like this, but I think most people will think of `ALTER USER` when referring to a user so it may be easier to follow.

Reply:
Maybe it's confusing I agree, I'll change it to user.